rr/actions/csv_logger.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+
+import csv
+
+
+class CSVLogger():
+    """
+    Class that logs events to a csv file
+
+    Methods
+    -------
+    write_header(csv_file : stream, fieldnames : list)
+        Wrtie the file headers
+    write_file(csv_file : str, fieldnames : list, results : dict)
+        Wrtie the inference results
+    """
+
+    def write_header(self, csv_file, fieldnames):
+        """ Sets the headers for the logging file
+
+        Parameters
+        ----------
+        csv_file : stream
+            The stream with the file data
+        fieldnames: list
+            The list with the fieldnames
+        """
+
+        csv_writer = csv.DictWriter(csv_file, fieldnames=fieldnames)
+        csv_writer.writeheader()
+
+    def write_file(self, csv_file, fieldnames, results):
```

Comment:
No need to pass in the csv_file nor the filenames. Please use the csv_writer you opened before.